src/subdomains/supporting/support-issue/support-issue.controller.ts

```diff
@@ -20,25 +24,39 @@ export class SupportIssueController {
   @Post()
   @ApiBearerAuth()
   @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
-  async createIssue(@GetJwt() jwt: JwtPayload, @Body() dto: CreateSupportIssueDto): Promise<void> {
+  async createIssue(@GetJwt() jwt: JwtPayload, @Body() dto: CreateSupportIssueDto): Promise<SupportIssueDto> {
     return this.supportIssueService.createIssue(jwt.account, dto);
   }
 
+  @Get()
+  @ApiBearerAuth()
+  @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
+  async getIssue(@GetJwt() jwt: JwtPayload, @Query() query: GetSupportIssueFilter): Promise<SupportIssueDto> {
+    return this.supportIssueService.getSupportIssue(jwt.account, query);
+  }
+
   @Post(':id/message')
   @ApiBearerAuth()
   @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
   async createSupportMessage(
     @GetJwt() jwt: JwtPayload,
     @Param('id') id: string,
     @Body() dto: CreateSupportMessageDto,
-  ): Promise<void> {
+  ): Promise<SupportMessageDto> {
     return this.supportIssueService.createSupportMessage(
       +id,
       [UserRole.SUPPORT, UserRole.ADMIN].includes(jwt.role) ? dto : { ...dto, author: CustomerAuthor },
       jwt.account,
     );
   }
 
+  @Get('/file')
+  @ApiBearerAuth()
+  @UseGuards(AuthGuard(), new RoleGuard(UserRole.ACCOUNT))
+  async getFile(@GetJwt() jwt: JwtPayload, @Query() query: GetSupportFile): Promise<BlobContent> {
+    return this.supportIssueService.getSupportIssueFile(jwt.account, query.name);
+  }
```

Comment:
I would probably use `:ticket-id/message/:message-id/file` as URL and omit the query